python/packages/azurefunctions/agent_framework_azurefunctions/_models.py

```diff
@@ -346,6 +350,8 @@ def to_dict(self) -> dict[str, Any]:
             result["correlationId"] = self.correlation_id
         if self.created_at:
             result["created_at"] = self.created_at
+        if self.orchestration_id:
+            result["orchestrationId"] = self.orchestration_id
```

Comment:
There seems to be some inconsistent naming here. The `result` keys above most use snake_case (e.g., `created_at`), but here you're using camelCase (`orchestrationId`).  Why do we have this inconsistency?